backend/content/models.py

```diff
@@ -25,23 +27,23 @@
 class Faq(models.Model):
     id = models.UUIDField(primary_key=True, default=uuid4, editable=False)
     org_id = models.ForeignKey("entities.Organization", on_delete=models.CASCADE)
-    name = models.CharField(max_length=255)
     question = models.TextField(max_length=500)
     answer = models.TextField(max_length=500)
     last_updated = models.DateTimeField(auto_now=True)
 
     def __str__(self) -> str:
-        return self.name
+        return self.question
 
 
 class Resource(models.Model):
     id = models.UUIDField(primary_key=True, default=uuid4, editable=False)
     name = models.CharField(max_length=255)
     description = models.TextField(max_length=500)
-    topics = ArrayField(models.CharField(max_length=255), default=list, blank=True)
+    topics = ArrayField(
```

Comment:
We'll likely need to move this to `resource_topics` at some point, but all of this is gonna be more clear once we start testing for it all :)